broken since v8.0.1

Hello,

I'm using this library at https://github.com/microlinkhq/ffprobe

I noted some of the last versions are missing; also, the latest version published is broken in some way I not fully understand.

Here is the way to reproduce the problem:

```
npm i lzma-native@8.0.5 && npm run postinstall

up to date in 874ms

> @microlink/ffprobe@1.0.2 postinstall
> node scripts/postinstall.js

dyld[52834]: missing symbol called
fish: Job 1, 'npm run postinstall' terminated by signal SIGABRT (Abort)

~/Projects/microlink/ffprobe master*
❯ npm i lzma-native@8.0.4 && npm run postinstall
npm ERR! code ETARGET
npm ERR! notarget No matching version found for lzma-native@8.0.4.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/kikobeats/.npm/_logs/2022-01-17T11_47_43_916Z-debug.log

~/Projects/microlink/ffprobe master*
❯ npm i lzma-native@8.0.3 && npm run postinstall
npm ERR! code ETARGET
npm ERR! notarget No matching version found for lzma-native@8.0.3.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/kikobeats/.npm/_logs/2022-01-17T11_47_50_145Z-debug.log

~/Projects/microlink/ffprobe master*
❯ npm i lzma-native@8.0.2 && npm run postinstall
npm ERR! code ETARGET
npm ERR! notarget No matching version found for lzma-native@8.0.2.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/kikobeats/.npm/_logs/2022-01-17T11_47_53_700Z-debug.log

~/Projects/microlink/ffprobe master*
❯ npm i lzma-native@8.0.1 && npm run postinstall

changed 1 package in 19s

> @microlink/ffprobe@1.0.2 postinstall
> node scripts/postinstall.js

# exit with 0 in this case, all fine
```

--- a/package.json
+++ b/package.json
@@ -33,7 +33,7 @@
   },
   "scripts": {
     "install": "node-gyp-build",
-    "prebuild": "prebuildify --napi --tag-armv",
+    "prebuild": "prebuildify --napi --electron-compat",
     "prepack": "[ $(ls prebuilds | wc -l) = '6' ] || (echo 'Some prebuilds are missing'; exit 1)",
     "test": "mocha --expose-gc -s 1000 -t 5000",
     "prepare": "npm run prepare-win32 || true",
@@ -48,6 +48,6 @@
     "bl": "^4.1.0",
     "jshint": "^2.12.0",
     "mocha": "^8.3.1",
-    "prebuildify": "^3.0.4"
+    "prebuildify": "^5.0.0"
   }
 }

